automatic/glmixer/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,19 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+  packageName            = 'glmixer'
+  fileType               = 'exe'
+  url                    = 'https://sourceforge.net/projects/glmixer/files/Windows%20Binary/GLMixer_1.5-1329_Windows.exe'
```

Comment:
@Redsandro I think I found a possible reason why you got that error,
when you tested the update you probably didn't have `-ChecksumFor 64` added at the end of `update.ps1` file.
Anyways, I've updated with a how I found it to work.